extensions/podman/packages/extension/src/podman-configuration.ts

```diff
@@ -30,24 +31,26 @@ const configurationRosetta = 'setting.rosetta';
  * Manages access to the containers.conf configuration file used to configure Podman
  */
 export class PodmanConfiguration {
+  mutex: Mutex | undefined;
```

Comment:
issue: initialize in the constructor and use private, it should not be accessible outside of this class